public/index.html

```diff
@@ -2,15 +2,13 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
```

Comment:
Are these changes necessary as well? Changing meta tags and such isn't bad or anything, but if we could keep PR's focused on only what they need to be, that'd make it easier to commit merges fast.

If these changes aren't necessary, it'd be best to not commit changes to index.html in this PR.